frontend/src/components/App/PluginSettings/PluginSettings.tsx

```diff
@@ -92,20 +93,17 @@ const EnableSwitch = (props: SwitchProps) => {
 
 /** PluginSettingsPure is the main component to where we render the plugin data. */
 export function PluginSettingsPure(props: PluginSettingsPureProps) {
-  const { t } = useTranslation(['translation']);
-
-  /** Plugin arr to be rendered to the page from prop data */
-  const pluginArr: any = props.plugins ? props.plugins : [];
+  const dispatch = useDispatch();
 
-  /** enableSave state enables the save button when changes are made to the plugin list */
-  const [enableSave, setEnableSave] = useState(false);
+  const { t } = useTranslation(['translation']);
 
-  /**
-   * pluginChanges state is the array of plugin data and any current changes made by the user to a plugin's "Enable" field via toggler.
-   * The name and origin fields are split for consistency.
+  /** Plugin arr to be rendered to the page from prop data
+   * Note:
+   * The pluginArr is the main array that will be used to render the plugin data to the page.
+   * Any changes made here is what will be rendered to the page.
    */
-  const [pluginChanges, setPluginChanges] = useState(() =>
-    pluginArr.map((plugin: PluginInfo) => {
+  const [pluginArr, setPluginArr] = useState<PluginInfo[]>(
```

Comment:
`plugins` would be more readable imho

Reply:
this comp already uses plugins as a prop which is always the redux plugins list from the state and since what we are using here is actually just a copy of that array I'm trying to keep it distinct

Reply:
maybe then `pendingPlugins` or `unsavedPlugins` or something like that, `arr` doesn't add anything to the name, it's an information about the type

Reply:
using the `pluginChanges` from before since its reworked now